userspace/libsinsp/CMakeLists.txt

```diff
@@ -201,7 +203,7 @@ if(NOT WIN32)
 					COMMENT "Generate CRI grpc code"
 					DEPENDS
 					COMMAND ${PROTOC} -I ${CMAKE_CURRENT_BINARY_DIR} --cpp_out=. ${CMAKE_CURRENT_BINARY_DIR}/cri.proto
-					COMMAND ${PROTOC} -I ${CMAKE_CURRENT_BINARY_DIR} --grpc_out=. --plugin=protoc-gen-grpc=${GRPC_CPP_PLUGIN} ${CMAKE_CURRENT_BINARY_DIR}/cri.proto
+					COMMAND LD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}:${PROTOBUF_SRC}/target/lib:${CARES_SRC}/target/lib ${PROTOC} -I ${CMAKE_CURRENT_BINARY_DIR} --grpc_out=. --plugin=protoc-gen-grpc=${GRPC_CPP_PLUGIN} ${CMAKE_CURRENT_BINARY_DIR}/cri.proto
```

Comment:
Ouch. Can we avoid messing with LD_LIBRARY_PATH here? Like e.g. with a static protoc/grpc build?

Reply:
The issue here is not protoc/grpc builds, but only the `grpc_cpp_plugin` that's being provided by the grpc cmake module when `USE_BUNDLED_GRPC` is on. Unfortunately, I have not found any suitable way to build it statically using the provided [grpc makefiles](https://github.com/falcosecurity/libs/blob/build/upgrade-deps/cmake/modules/grpc.cmake#L66).
So the workaround was this hacky thing.

Btw, [building gRPC with make](https://github.com/grpc/grpc/blob/master/BUILDING.md#building-with-make-on-unix-systems-deprecated) is not recommended anymore. However, we are still stuck with that. Unless we want to completely review the way we build grpc (maybe in the future?), I don't see any other solution (I might be wrong, ofc).

Anyway, it may worth to comment that line to explain the workaround. Wdyt?